util/withAttr.js

```diff
@@ -1,7 +1,9 @@
 "use strict"
 
-module.exports = function(attrName, callback, context) {
+module.exports = function(attrName, callback, context, shouldRedraw) {
+	if(shouldRedraw === undefined) shouldRedraw = true;
 	return function(e) {
+		e.redraw = shouldRedraw;
```

Comment:
Personally I'd prefer `e.redraw = (shouldRedraw ? true : false)` for added readability, but that's probably mincing words.